src/qibo/backends/__init__.py

```diff
@@ -147,24 +181,32 @@ def create(self, dtype):
 
 
 def get_backend():
-    return str(GlobalBackend())
+    return str(_Global.get_backend())
```

Comment:
```suggestion
    return str(_Global.backend())
```
should be alright.

You're probably asking "which backend would I use if I were to execute a circuit now?", and the answer is `_Global.backend()`, by definition.

Reply:
You could even add something like this as documentation.

Reply:
Updated this.